lib/monobank/client.rb

```diff
@@ -11,15 +12,21 @@ def bank_currency
     end
 
     def client_info(token:)
-      Personal::ClientInfo.new(token: token).call
+      Personal::ClientInfo.new(auth: auth(token:)).call
     end
 
     def statement(token:, account_id:, from:, to: nil)
-      Personal::Statement.new(token: token, account_id: account_id, from: from, to: to).call
+      Personal::Statement.new(account_id: account_id, from: from, to: to, auth: auth(token:)).call
     end
 
     def set_webhook(token:, url:)
-      Personal::Webhook.new(token: token, url: url).call
+      Personal::Webhook.new(url: url, auth: auth(token:)).call
+    end
+
+    private
+
+    def auth(token:)
+      Auth::Private.new(token: token)
```

Comment:
I noticed the shorthand syntax like `auth(token:)` in the PR. While it's concise, its clarity might be reduced for those unfamiliar with this Ruby syntax, especially when the argument isn't directly visible.

For consistency, if we're using this approach, should we apply it uniformly across our code, such as with `Auth::Private.new(token:)`?

Reply:
yup, makes sense. unified the approach in this particular file